docs/source/endpoints/upgrade.md

```diff
@@ -0,0 +1,122 @@
+---
+myst:
+  html_meta:
+    "description": "The @upgrade endpoint exposes  Plone backend."
+    "property=og:description": "The @system endpoint exposes system information about the Plone backend."
+    "property=og:title": "Upgrade"
+    "keywords": "Plone, plone.restapi, REST, API, Upgrade"
+---
+
+(upgrade)=
+
+# Upgrade
+
+A Plone site needs to be in sync with the version available on the file system.
+The `@upgrade` endpoint exposes upgrade information about the Plone backend, and supports running the upgrade of the site.
+
+```{note}
+The upgrade endpoint is protected by the `cmf.ManagePortal` permission that requires the Site Administrator or Manager role.
```

Comment:
I don't think the Site Administrator role has "Manage portal". That's one of the key differences between Site Administrator and Manager.

Reply:
Fixed